_posts/0000-01-02-Newton02.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
then I still looked for how to program without using this platform. Actually, I realize I can do better if I learn how to use it that why I choose to spend my time in learning how to use Github.